docs/integrations/web-servers/apache-tomcat.md

```diff
@@ -612,11 +610,11 @@ Log format description: [https://stackoverflow.com/questions/4468546/explanation
     * environment = <Environment_Name> #such as Dev, QA or Prod.
     ```
 1. Configure the **Advanced** section:
-* **Enable Timestamp Parsing.** Select Extract timestamp information from log file entries.
-* **Time Zone.** Choose the option, **Ignore time zone from log file and instead use**, and then select your Tomcat Server’s time zone.
-* **Timestamp Format.** The timestamp format is automatically detected. **Encoding.** Select UTF-8 (Default).
-* **Enable Multiline Processing.** Detect messages spanning multiple lines
-    * Infer Boundaries - Detect message boundaries automatically
+   * **Enable Timestamp Parsing.** Select Extract timestamp information from log file entries.
+   * **Time Zone.** Choose the option, **Ignore time zone from log file and instead use**, and then select your Tomcat Server’s time zone.
+   * **Timestamp Format.** The timestamp format is automatically detected. **Encoding.** Select UTF-8 (Default).
+   * **Enable Multiline Processing.** Detect messages spanning multiple lines
+   * Infer Boundaries - Detect message boundaries automatically
 1. Click **Save**.
 
 At this point, Tomcat logs should start flowing into Sumo Logic.
```

Comment:
It's out of scope for this PR, since you're only looking for asterisks problems. But the Docusaurus 3 upgrade also messed up indenting in the following numbered lists in this article:
https://help.sumologic.com/docs/integrations/web-servers/apache-tomcat#step-1-configure-metrics-collection
https://help.sumologic.com/docs/integrations/web-servers/apache-tomcat/#step-2-configure-logs-collection

Reply:
thank you! I have another PR coming (in progress) specifically for indentation